cmd/update.go(Ignore)

```diff
@@ -0,0 +1,29 @@
+package cmd
```

Comment:
Please remove this file. It's totally fine if you want to propose a new feature... but please do it in another PR.